packages/dev/core/src/ShadersWGSL/iblShadowsIcdfy.fragment.fx

```diff
@@ -0,0 +1,41 @@
+#define PI 3.1415927
+varying vUV: vec2f;
+
+var cdfy: texture_2d<f32>;
+
+fn fetchCDF(y: u32, invocationId: u32) -> f32 {
+    return textureLoad(cdfy,  vec2u(invocationId, y), 0).x;
+}
+
+fn bisect(size: u32, targetValue: f32, invocationId: u32) -> f32
+{
+    var a: u32 = 0;
+    var b = size - 1;
+    while (b - a > 1) {
+        var c = (a + b) >> 1;
+        if (fetchCDF(c, invocationId) < targetValue) {
+            a = c;
+        }
+        else {
+            b = c;
+        }
+    }
+    return mix( f32(a),  f32(b), (targetValue - fetchCDF(a, invocationId)) / (fetchCDF(b, invocationId) - fetchCDF(a,invocationId))) /  f32(size - 1);
+}
+
+@fragment
+fn main(input: FragmentInputs) -> FragmentOutputs {
+    var cdfSize: vec2u = textureDimensions(cdfy, 0);
+    var cdfHeight: u32 = cdfSize.y;
+    var currentPixel: vec2u =  vec2u(input.position.xy);
```

Comment:
shader preprocessor can't handle this expression see [here](https://github.com/BabylonJS/Babylon.js/blob/master/packages/dev/core/src/Engines/WebGPU/webgpuShaderProcessorsWGSL.ts#L280)
```suggestion
    var currentPixel: vec2u =  vec2u(fragmentInputs.position.xy);
```